packages/manager/modules/pci/src/projects/project/storages/blocks/add/add.html

```diff
@@ -0,0 +1,95 @@
+<div class="p-5">
+    <oui-back-button
+        data-on-click="$ctrl.back()">{{:: 'pci_projects_project_storages_blocks_add_back_label' | translate }}</oui-back-button>
```

Comment:
Prefer using `translate` directive. `{{` notation isn't the chosen one for all parts.